etc/api/angular_devkit/core/src/_golden-api.d.ts

```diff
@@ -1259,6 +1260,12 @@ export interface WorkspaceProject {
     targets?: WorkspaceTool;
 }
 
+export interface WorkspaceProjectI18N {
+    locales: Record<string, string>;
+    missingTranslation?: 'warning' | 'error' | 'ignore';
+    sourceLocale?: string;
```

Comment:
What is the purpose of this property?

Reply:
//cc @clydin

Reply:
I am guessing that it is used to generate the output path for the default bundle.

Reply:
It will have three uses:
* value for the output path, as mentioned
* value for lang html attribute in the index
* signifies $localize content should be used verbatim when inlining

A future fourth use will be automatic registration of application locale data.